inst/report_downloads/reportDocx.Rmd

```diff
@@ -143,11 +148,11 @@ if('Maintenance Metrics' %in% params$report_includes){
       )
     ) %>%
     dplyr::select(`Metric Name`, `Metric Description`, `Metric Value`, `Metric Score`) %>%
-    knitr::kable(format = 'pandoc')
+    kableExtra::kbl(format = 'pipe', booktabs = T) 
 }
 ```
 
-`r if ('Maintenance Metrics' %in% params$report_includes) {"* Metrics whose score is NA will not impact the package {riskmetric} score"}`
+`r if ('Maintenance Metrics' %in% params$report_includes) {"Metrics whose score is NA will not impact the package {riskmetric} score"}`
```

Comment:
If we are going to remove `*` can we at least do something to make it clear this is a footer to the table.

Reply:
asterisk is back in the footnote